pkg/errors.go

```diff
@@ -16,8 +16,11 @@ type isNotExister interface {
 // IsNotExist returns a boolean indicating whether the error is known to report that
 // a path does not exist.
 func IsNotExist(err error) bool {
-	e, ok := err.(isNotExister)
-	return ok && e.isNotExist()
+	var e isNotExister
```

Comment:
Why the change here?

Reply:
Because of the use of `fmt.Errorf`, the error is now wrapped. I want `IsNotExist` to still be able to recognized a wrapped error.